website/docs/troubleshooting.md

```diff
@@ -2,6 +2,23 @@
 title: Troubleshooting
 ---
 
+## Copa and Trivy throw errors when Oracle Linux is passed in
+
+Copa supports patching Oracle Linux in two ways:
+
+With a vulnerability scan, `--ignore-errors` must be passed in:
+
+```bash
+patch -r /oracle-7.9-vulns.json -i docker.io/library/oraclelinux:7.9 --ignore-errors
+```
+
+Without a vulnerability scan, Copa will update all packages in the image:
+
+```bash
+patch -i docker.io/library/oraclelinux:7.9
```

Comment:
copa patch?

Reply:
This would be referencing `copa patch`

Reply:
lets change it to copa patch to be consistent with other parts of the doc and avoid confusion